contrib/__init__.py

```diff
@@ -0,0 +1,6 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
```

Comment:
I don't think it makes sense to declare `contrib` to be a Python package. The scripts in this directory are not meant to be imported, and the existing subpackages expect to be top-level packages. I guess this breaks other stuff.

Reply:
That's true. In that case I don't know how to make `nose.collector` find the tests for mergeBenchmarkSets though.
I found this in the docs[(source)](https://nose.readthedocs.io/en/latest/finding_tests.html):

> Directories that don’t look like tests and aren’t packages are not inspected.

Do you know how to change this?

Reply:
Not right now, but the docs says that this is only the default and that "that plugins and command line options can change any of those rules." I suspect there is a way to additionally add `contrib` to the list of directories where it should search for tests.

Reply:
Found it: The option is `--include contrib`. Added to setup.cfg.